molecule/default/Dockerfile.j2

```diff
@@ -7,9 +7,9 @@ FROM {{ item.registry.url }}/{{ item.image }}
 FROM {{ item.image }}
 {% endif %}
 
-RUN if [ $(command -v apt-get) ]; then apt-get update && apt-get install -y python sudo bash ca-certificates && apt-get clean; \
```

Comment:
Is this required or an accidental change? It does not seem to affect our CI.

Reply:
yes not intentional! Will fix this

Reply:
Fixed. I was a bit heavy handed with the `sed` commands!